### ISSUE TITLE
manualRowMove not changing the data

### ISSUE THREAD

**Issue by sunpc** [handsontable/vue-handsontable-official#59]
Hello,
I'm using the latest vue version (2.0.0) with handsontable CE 4.0.0, and found an issue that the data is not being changed after manual moved the data from the handsontable.

The HotTable is defined as below:
`<HotTable :root="root" :settings="hotSettings"></HotTable>`

In hotSettings, I've put manualRowMove: true inside:
`this.hotLayoutSettings = {
          data: this.data, 
          rowHeaders: true,   
          colHeaders: ['Name', 'Type', 'Length'],  
          autoWrapRow: false,  
          contextMenu: {      
            items: {
              "row_above": {},
              "row_below": {},
              "hsep1": "---------",
              "remove_row": {}
            }
          },
          fillHandle: "vertical", 
          fixedColumnsLeft: 0, 
          fixedRowsTop: 0,       
          manualRowMove: true,
          manualColumnResize: true, 
          minSpareRows: 1,          
          columns: [                
            {
              data: 'name',
              type: 'text',
              allowEmpty: false
            },
            {
              data: 'type',
              type: 'dropdown',
              source: ['US', 'CA', 'JP'],
              strict: true, 
              allowEmpty: false
            },
            {
              data: 'length',
              type: 'numeric',   
              numericFormat: {
                pattern: '0'
              },              
              allowEmpty: false
            }
          ],
          stretchH: 'all'
        };
      }`

However, after moved the data from the webpage, this.data was not changed - only UI was changed.

Would you please help to advise what's wrong with this? Thanks so much!!

**Comment by AMBudnik** [comment]
Hi @sunpc 

please put up an example and step by step scenario

**Comment by AMBudnik** [comment]
I am closing the issue as there's no feedback from the Author

**Comment by wangqiffm** [comment]
@AMBudnik 
I also have same problem.

handsontable@6.0.1
handsontable/vue@3.0.0

demo:
https://jsfiddle.net/f1aqvch7/8/

use context menu "remove row" of handsontable to remove one row
the second table doesn't update itself

**Comment by AMBudnik** [comment]
the case is a bit different but thank you for sharing. We will check that.

**Comment by wojciechczerniak** [comment]
TLDR; In Vue properties passed to component should be binded only one way. Use events / hooks to alter your data.

----
`manualRowMove` does not change `data` object. In both cases `data` shouldn't be referenced anywhere in your code, it's for internal use by Handsontable. Some operations modify the `data` and some don't (they are kept as transactions).

To render table based on Handsontable `data` or any other operations always use hooks that Handsontable provides: `afterChange`, `afterRemoveRow`, `afterRowMove` etc. to name just few. 

General idea is covered in #103.
